db/seeds.rb

```diff
@@ -39,63 +23,64 @@
   r.time_zone = 'America/Los_Angeles'
 end.save!
 
-primary_location = Location.find_by(city_name: ENV['PRIMARY_CITY_NAME'])
-Widget.find_or_initialize_by(
-  name: "Guests"
-) do |r|
-  r.enabled = false
-  r.duration_seconds = 20
-  r.position = 0
-  r.sidebar_text = 'Guests'
-  r.show_weather = false
-  r.location = primary_location
-end.save!
+Location.all.each do |location|
+  Widget.find_or_initialize_by(
+    name: "Guests",
+    location: location
+  ) do |r|
+    r.enabled = false
+    r.duration_seconds = 20
+    r.position = 0
+    r.sidebar_text = 'Guests'
+    r.show_weather = false
+  end.save!
```

Comment:
What does line 29 do and the following values you changed from the original version?

Reply:
Widgets are being created for each location instead of just the primary location. There is a whitespace change happening also that makes the diff a little more confusing.

---
Protip:

> Append ?w=1 to the github diff url.
>
> This can be used on pull requests, compare pages etc

https://coderwall.com/p/iuvlpg/remove-whitespace-changes-from-diff-in-github